### DISCUSSION TITLE
How can I exclude crate rfd from the build?

### DISCUSSION THREAD

**Discussion by derekmahar** [dathere/qsv#2155] [Q&A] [answered]
How can I exclude crate [rfd](https://github.com/PolyMeilex/rfd) from the qsv build?  Crate rfd breaks my qsv build on a headless server that has no window manager because it indirectly requires crate [wayland-sys](https://github.com/smithay/wayland-rs) which fails to build because wayland-sys cannot find wayland-client which isn't installed.

**Comment by jqnatividad** [comment, marked as answer]
Will have to gate the `prompt` command, which uses `rfd`, behind a feature.

This should be available in the next release.

**Reply by jqnatividad** [reply]
`prompt` feature implemented in #2163
